src/main/ngapp/src/app/register/register.component.html

```diff
@@ -99,10 +13,10 @@ <h4>Registration</h4>
         <input nbInput type="text" name="firstName" ngModel #firstName="ngModel" placeholder="First name" required [status]="firstName.dirty? (firstName.invalid? 'danger': 'success'): 'basic'">
         <input nbInput type="text" name="lastName" ngModel #lastName="ngModel" placeholder="Last name" required [status]="lastName.dirty? (lastName.invalid? 'danger': 'success'): 'basic'">
         <input nbInput type="email" name="email" email ngModel #userEmail="ngModel" placeholder="Email address" required [status]="userEmail.dirty? (userEmail.invalid? 'danger': 'success'): 'basic'">
-        <nb-select placeholder="Select Showcase" ngModel #language="ngModel" name="language" id="language" required [status]="language.dirty? (language.invalid? 'danger': 'success'): 'basic'">
+        <nb-select placeholder="Select Language" ngModel #language="ngModel" name="language" id="language" required [status]="language.dirty? (language.invalid? 'danger': 'success'): 'basic'">
```

Comment:
from testing, when a user logs out and directly tries to register a new user, the application hangs here, and the dropdown menu fails to show up. when i refreshed the page, this problem disappeared and i was able to select language. what could be the problem?

Reply:
Yeah. I noticed that as well.

Did a bit of research into this and found that it has something to do with the Nebular layout. Apparently we have to wrap the contents in a layout. 
Because we are going to deprecate the form, I didn't wrap it in a layout column. I did however, wrap all other instances of the select in a layout column.